mlocal/frags/Makefile.stub

```diff
@@ -67,6 +72,15 @@ $(singularity_INSTALL): $(singularity)
 	$(V)install -d $(@D)
 	$(V)install -m 0755 $(singularity) $(singularity_INSTALL) # set cp to install
 
+# runsy
+$(runsy): $(go_OBJ) $(singularity_OBJ)
```

Comment:
This isn't quite correct. As you can see, `singularity_OBJ` refers to the dependencies of `src/cmd/singularity/cli.go`. I think you want to introduce a variable above called `runsy_OBJ := $(shell $(SOURCEDIR)/makeit/gengodep $(SOURCEDIR)/src/cmd/runsy/cli.go)` instead.